sql/Pg-database.sql

```diff
@@ -743,6 +758,18 @@ CREATE TABLE person (
  );
 
 COMMENT ON TABLE person IS $$ Every person, must have an entity to derive a common or display name. The correct way to get class information on a person would be person.entity_id->entity_class_to_entity.entity_id. $$;
+-- Software robot. Currently implemented as a degraded person
+CREATE TABLE robot (
+    id serial PRIMARY KEY,
+    entity_id integer references entity(id) not null,
+    first_name text check (first_name ~ '[[:alnum:] _\-\.\*]?'),
+    middle_name text,
+    last_name text check (last_name ~ '[[:alnum:] _\-\.\*]') NOT NULL,
+    created date not null default current_date,
+    unique(entity_id)
+ );
+
+COMMENT ON TABLE robot IS $$ Every robot, must have an entity to derive a common or display name. The correct way to get class information on a robot would be robot.entity_id->entity_class_to_entity.entity_id. $$;
```

Comment:
Here too; basically, we don't want to change Pg-database for a (long?) while. It's a DDL change, so, goes into sql/changes/.